tests/CSharp/CSharp.h

```diff
@@ -1203,4 +1203,7 @@ class DLL_API TestIndexedProperties
 int TestIndexedProperties::operator[](const int& key)
 {
     return key;
-}
\ No newline at end of file
+}
+
+struct SomeStruct {};
+extern const SomeStruct SomeArray[];
```

Comment:
As I said, name after what you test. For example, you could christen it "ArrayOfVariableSize".